src/pages/RFOX/RFOX.tsx

```diff
@@ -12,16 +15,26 @@ const direction: StackDirection = { base: 'column-reverse', xl: 'row' }
 const maxWidth = { base: 'full', lg: 'full', xl: 'sm' }
 const paddingVerticalResponsiveProps = { base: 8, md: 16 }
 
+const stakingAssetId = foxOnArbitrumOneAssetId
```

Comment:
I don't get why we would need to put this assetId at this level I hate the fact that we have to pass the asset id through props, it doesn't look composable to me because if we need to use the overview somewhere else, outside of this particular component, inside another quick view component example in another place, we would need to bring back this asset id

Wdyt about adding another hook like `useRfoxAsset` returning `{ stakingAssetId, stakingAssetAccountId, stakingAssetAccountAddress }`

Consume this hook directly inside any components needing this assetId and remove the props, we remove any dependencies between parents and childrens

Reply:
So as far as composibility goes here, we're only concerned as making things composable in the context of RFOX AFAIK, since this has quite direct ties to it.
We're always going to need a `stakingAssetId`, meaning it has to come from up top.

re: `useRfoxAsset`, how would the API look like for it? i.e it would need to take a `stakingAssetId` either way (which would have to be passed then), and as far as returning an AccountId, we couldn't do that because of multi-account, the fact that we only support account 0 is a virtual limitation for launch, but we effectively support multi-account, simply restrict it.

Reply:
Discussed this one out in more details, conceptually like the idea of having a context vs. passing things down, though not a now thing